spec/unit/mutant/actor/binding_spec.rb

```diff
@@ -1,12 +1,13 @@
 RSpec.describe Mutant::Actor::Binding do
-  let(:actor_a)    { double('Actor-A', sender: sender_a, receiver: receiver_a) }
-  let(:sender_a)   { double('Sender-A')                                        }
-  let(:sender_b)   { double('Sender-B')                                        }
-  let(:receiver_a) { double('Receiver-A')                                      }
-  let(:payload)    { double('Payload')                                         }
-  let(:type)       { double('Type')                                            }
+  let_instance(:sender_a,   Mutant::Actor::Sender                                           )
+  let_instance(:sender_b,   Mutant::Actor::Sender                                           )
+  let_instance(:mailbox,    Mutant::Actor::Mailbox, receiver: :receiver_a, sender: :sender_a)
```

Comment:
@mbj if you can do 99% of what you need with the syntax you came up with I guess that's good.

My main reason for flagging it is that you have a specific context where you could simply refer to things like `receiver_a` without having to do any special meta-programming. It was a red flag to me that extra logic is needed to work-around it.